cmd/kube-proxy/app/server.go

```diff
@@ -142,66 +142,101 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&o.ConfigFile, "config", o.ConfigFile, "The path to the configuration file.")
```

Comment:
Nowadays the deprecated flags are not showing up at all in the help. This `flagset` is missing something like:

```
fs := pflag.NewFlagSet("", pflag.ExitOnError)
defer func() {
		// Unhide deprecated flags. We want deprecated flags to show in Kube-proxy help.
		// We have some hidden flags, but we might as well unhide these when they are deprecated,
		// as silently deprecating and removing (even hidden) things is unkind to people who use them.
		fs.VisitAll(func(f *pflag.Flag) {
			if len(f.Deprecated) > 0 {
				f.Hidden = false
			}
		})
		mainfs.AddFlagSet(fs)
	}()
```

Reply:
Cool, thanks for the suggestions, I think we'll do it in the future when we finally need to deprecate them!